berkdb/lock/lock.c

```diff
@@ -3947,53 +3947,38 @@ __lock_getlocker(lt, locker, indx, retries, flags, retp)
 	u_int32_t flags;
 	DB_LOCKER **retp;
 {
-	int ret;
 	int created;
-	DB_LOCKREGION *region;
-	u_int32_t partition;
-
-	int create = LF_ISSET(GETLOCKER_CREATE);
+	int lk_create = LF_ISSET(GETLOCKER_CREATE);
 	int is_logical = LF_ISSET(GETLOCKER_LOGICAL);
 	u_int32_t keep_part_lock = LF_ISSET(GETLOCKER_KEEP_PART);
 	u_int32_t get_locker_lock = !LF_ISSET(GETLOCKER_DONT_LOCK);
 
-	region = lt->reginfo.primary;
-	partition = locker % gbl_lkr_parts;
+	DB_LOCKREGION *region = lt->reginfo.primary;
+	u_int32_t partition = locker % gbl_lkr_parts;
 
 	lock_locker_partition(region, partition);
 
-	ret = __lock_getlocker_int(lt, locker, indx, partition, create,
-	    retries, retp, &created, is_logical);
-
-	if (ret || *retp == NULL || keep_part_lock == 0) {
+	int ret = __lock_getlocker_int(lt, locker, indx, partition, lk_create,
```

Comment:
Berkeley DB code is K&R C. Let's adhere to K&R unless we have a good reason not to.